OWNERS_ALIASES

```diff
@@ -6,3 +6,6 @@ aliases:
     - ahmedwaleedmalik
     - khizerrehan
     - waseem826
+
+  sig-cluster-management: []
```

Comment:
Do we really need sig-cluster-management here?

Reply:
This was just to silence the prow warnings, in actuality we don't need this SIG here, but I didn't want to go through all owners files yet 😁